cli/internal/otel/receiver.go

```diff
@@ -255,12 +257,12 @@ func StartOtelReceiver(ctx context.Context, opts ...OtelReceiverOption) (*OtelRe
 
 	factory := otlpreceiver.NewFactory()
 	config := factory.CreateDefaultConfig().(*otlpreceiver.Config)
-	config.GRPC = nil
+	config.GRPC = configoptional.None[configgrpc.ServerConfig]()
 	port, err := getFreePort()
 	if err != nil {
 		return nil, err
 	}
-	config.HTTP.Endpoint = fmt.Sprintf("localhost:%d", port)
+	config.HTTP.Get().ServerConfig.Endpoint = fmt.Sprintf("localhost:%d", port)
```

Comment:
I tried to run a sync with these changes and the `--tables-metrics-location` flag (https://cli-docs.cloudquery.io/docs/reference/cli/cloudquery_sync#options) and the code panics since `.Get()` returns `nil`. Looking into it